### DISCUSSION TITLE
How to disable Google Fonts in Quarto website

### DISCUSSION THREAD

**Discussion by cbrnr** [quarto-dev/quarto-cli#2682] [Show and Tell]
I thought that maybe some of you might be interested in disabling Google Fonts from your Quarto websites (because of data privacy regulations or to speed up loading times a little), so I have written a short summary on how to do that:

https://cbrnr.github.io/blog/removing-google-fonts/

**Comment by mcanouil** [comment]
Thanks for this tip!
If I may the title is a bit misleading about what it is actually about, which is "how to avoid web requests to retrieve online fonts in Quarto website/documents?"

**Reply by cbrnr** [reply]
True, if you equate Google Fonts with the actual fonts. I was thinking of it as the web service, which gets removed. But yes, the fonts stay of course. Can you think of a shorter version?
